src/app/fdctl/run/tiles/fd_pack.c

```diff
@@ -0,0 +1,292 @@
+#include "tiles.h"
+
+/* fd_pack provides services to packlicate multiple streams of input
+   fragments and present them to a mix of reliable and unreliable
+   consumers as though they were generated by a single multi-stream
+   producer.
```

Comment:
The word `packlicate` made me laugh out loud, but this comment needs to be updated